python/tests/material_dispersion.py

```diff
@@ -13,12 +13,17 @@ def setUp(self):
             mp.LorentzianSusceptibility(frequency=0.5, gamma=0.1, sigma=2e-5)
         ]
 
-        default_material = mp.Medium(epsilon=2.25, E_susceptibilities=susceptibilities)
+        self.default_material = mp.Medium(epsilon=2.25, E_susceptibilities=susceptibilities)
+
```

Comment:
Set the default material to air so that we are sure that the susceptibility comes from the user function

Reply:
I removed the old version.